### DISCUSSION TITLE
Cooling mode for Homematic IP Thermostats (or CCU3 groups) in home assistant

### DISCUSSION THREAD

**Discussion by dlbogdan** [SukramJ/aiohomematic#1410] [Q&A]
Hi,
I would like to start by saying that I'm very thankful that HomeMatic exists, and even more thankful for your work bringing Homematic IP into homeassistant. I've been searching for a while now for something that has some local logic, like PLCs have, so that in the event that the network infrastructure goes down, or the hub is frozen, simple logic would still work.

So I've bought a house, and then I've bought 20 eTRVs (eTRV-2 and eTRV-B-2 types), and 5 with 5 more coming Wall Thermostats (WTH-1 and WTH-A types). I also bought a WHS-2 switch. The CCU is a raspberry pi 3 with an rf-mod hat, running the latest RaspberryMatic software. 

The main architecture is simple: I have a gas boiler (actually two but I'll keep it simple for now), that is switched on/off by the WHS-2 
I then have multiple zones that are configured as groups in RaspberryMatic as HmIP-Heating Control, that consists of multiple eTRVs mounted on radiators and a wall thermostat. 
Some rooms only have the eTRVs on the radiators. 
First problem: How would I command the WHS-2 switch to be on when ANY of the groups and individual eTRVs would demand heating, other than with Programs & Raspmatic connections, which works fine, but only as long as the raspberrymatic device is functional (i.e. not frozen or faulted)

2nd problem: I also have two AC units, that I would like to set their cool mode in the summer by the use of the same wall thermostats.
I've tried a lot in homeassistant and played with a lot of settings also on raspberrymatic side for WEEKS but I can't have the generic thermostat card show me also a COOLING mode or HEAT/COOL combo. Only Auto/Heat/Off. 

Other than a janky home assistant automation that would just check a helper to see if I set it to Summer mode, and then maybe read the value off of the entity thermostat when it is changed, and set the temperature setpoint to the AC unit with a template (just thinking, haven't done this), how would one solve this more elegantly ?

Thank you in advance!

**Comment by dlbogdan** [comment]
yuhu! Exciting! How is this resolved @SukramJ ?  Any links ?
Thanks a lot!

**Reply by SukramJ** [reply]
Sorry, i closed this due to inactivity.

HEATING_COOLING entity is avalable for relevant devices since 1.65.0
